src/streamlit_extras/altex/__init__.py

```diff
@@ -445,31 +464,31 @@ def example_bar_grouped():
     )
 
 
-__func__ = chart
+__funcs__ = [_chart]
```

Comment:
Didn't actually add all functions there since I'm using partial and partial doesn't inherit the __doc__ attribute. I'm too lazy to have specific docstrings for all of them... Only exposing the core function which is _chart!

Reply:
That's fine -- I spent *way* too long trying to get this to work properly. I almost got it, but ultimately couldn't fix the source code view, so I gave up. I wonder if it's worth rewriting this someday to make these not be partials, but I think it's fine for now since all the examples show you how to use them.

Reply:
Yea I mean, see how I hacked a new `_partial` just to inherit the `__name__` attribute haha. Would love to use something else than partial which kinda inherits. Well now I realize I could just do that have e.g. `line_chart.__doc__ = _chart.__doc__.replace("chart", "line chart")`... but yea let's stick to that for now, especially since I'm probably the only user of altex haha